src/plugins/ui_feature/collaborative.ts

```diff
@@ -32,8 +42,53 @@ export class CollaborativePlugin extends UIPlugin {
     );
   }
 
-  getClient() {
-    return this.session.getClient();
+  handle(cmd: Command) {
+    switch (cmd.type) {
+      case "JUMP_TO_CLIENT":
+        this.jumpToClient(cmd.clientId);
+        break;
+      case "SHOW_CLIENT_TAG":
+        this.showClientTag = true;
+        break;
+      case "HIDE_CLIENT_TAG":
+        this.showClientTag = false;
+        break;
+    }
+  }
+
+  jumpToClient(clientId: ClientId) {
+    const clients = this.getConnectedClients();
```

Comment:
instead of looping over all clients
```suggestion
    const client = this.getClient(clientId);
    if (client.position) {
```